.github/workflows/build.yml

```
@@ -149,8 +149,11 @@ jobs:
         security import certificate.p12 -k build.keychain -P $MACOS_CERTIFICATE_PWD -T /usr/bin/codesign
         security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k Techminohaowan build.keychain
         /usr/bin/codesign --force --deep -s 79B81FC5EA155243C973B5417B0996501F00EF55 ./Techmino.app -v
+    - name: Pack Techmino
+      run: |
+        zip -r -y Techmino-Packed.app.zip Techmino.app
     - name: Upload
       uses: actions/upload-artifact@v2
       with:
         name: Techmino_${{ env.Version }}_${{ env.CommitID }}_macOS
-        path: Techmino-Packed.app
\ No newline at end of file
+        path: Techmino-Packed.app.zip
```

现在无法上传的原因只是路径写错了（
_而对于GitHub打包单个文件夹会拆一层这个问题_

如果你不介意的话
或许可以
```suggestion
    - name: Create Upload Folder
      run: |
        mkdir Upload
        mv Techmino.app Upload
    - name: Upload
      uses: actions/upload-artifact@v2
      with:
        name: Techmino_${{ env.Version }}_${{ env.CommitID }}_macOS
        path: Upload
```

---

我还是不确定GitHub上传文件夹的时候symlink会不会出问题，你可以试试（